### PR TITLE
Railgun S ED

### PR DESCRIPTION
**Pull request by SeraphJACK**

### CODE REVIEW

File: album/1000393018.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[[discs.tracks]]
+title = "Grow Slowly"
+tags = ["アニメ「とある科学の超電磁砲S」", "ED"]
+
+[[discs.tracks]]
+title = "stand still"
+
+[[discs.tracks]]
+title = "Grow Slowly (instrumental)"
```
Comment (Yesterday17) [inline album/1000393018.toml:21]:
```suggestion
title = "Grow Slowly（Instrumental）"
```

File: album/1000393018.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+type = "instrumental"
+tags = ["アニメ「とある科学の超電磁砲S」", "ED"]
+
+[[discs.tracks]]
+title = "stand still (instrumental)"
```
Comment (Yesterday17) [inline album/1000393018.toml:26]:
```suggestion
title = "stand still（Instrumental）"
```

File: album/1000393018.toml (hunk 1 above)
Comment (Yesterday17) [reply album/1000393018.toml:21]: https://iguchiyuka.jp/disco_data/single/20130515_single.html

File: album/1000393018.toml (hunk 2 above)
Comment (Yesterday17) [reply album/1000393018.toml:26]: 顺带 anime 盘也收一下（

**Review by Yesterday17 — APPROVED** [review]

**Review by tuzi3040 — APPROVED** [review]

**Pull request merged**